src/driver/aurora-data-api-pg/AuroraDataApiPostgresQueryRunner.ts

```diff
@@ -29,6 +29,8 @@ export class AuroraDataApiPostgresQueryRunner extends PostgresQueryRunnerWrapper
      */
     driver: AuroraDataApiPostgresDriver;
 
+    client: any;
```

Comment:
Can this be private?

Reply:
I just made it the same level of visibility as all the other properties. If I had to I'd do protected to allow access to everyone who wants to extend the class.

Reply:
The other property, driver, is on every query runner.

Well, they can't code against it if we make it private, right?  That's some of the use of visibility because it means you know what is and isn't a public API that can or can't change.

Is it possible that we can make it private and then if it's needed we can loosen up the protections on it then?

Or are you saying that there are already implementations that expect this to be available?

Reply:
I made them protected. I think it's the best way forward because all the other properties there seems to have it, it makes it non-public and also allows extensibility. Also I don't think it's good making it private because the change if the need arises will take some time because people will need to wait for the release of a new version of the ORM. I also don't like making it possible for people to do their thing and if they decide to extend the class they probably know what they're doing.